lib/pubsub/index.js

```diff
@@ -13,8 +16,14 @@ function PubSub(options) {
   // isn't complete until the callback returns from Redis
   // Maps channel -> true
   this.subscribed = {};
+
+  var self = this;
```

Comment:
As a style convention, the use of `self` as a variable totally works and that is what I use when there is not a better name to use. (You'll see this in sharedb tests even)

However, in sharedb and other repos, we do prefer to use the instance variable name (`pubSub` would be the instance name of the class `PubSub`) rather than `self`. I believe this makes the code and stack traces more specific, easier to understand, and easier to grep / search.